### PR TITLE
User story#3

### PR DESCRIPTION
**Pull request by pmarx92**
I had to made some changes to the _app.js, so that all pages can access the useState. Maybe someone could take a look at this one, im not sure if it is correct the way i shared the useState.

Everything else is just to show the Card inside the list page.

#3

### CODE REVIEW

File: pages/Add/index.js [hunk 1]
```diff
@@ -27,16 +16,11 @@ export default function Add() {
       fishweight: +e.target.weight.value,
```
Comment (jecaps) [inline pages/Add/index.js:16]: Everything looks good! Just a small suggestion: use camelCase or snake_case in naming the object keys. 😁

Comment (pmarx92) [reply pages/Add/index.js:16]: ah, true that one! Thanks :)

File: pages/Add/index.js [hunk 2]
```diff
@@ -27,16 +16,11 @@ export default function Add() {
...
       fishlength: +e.target.length.value,
       location: e.target.location.value,
-      date: startDate,
+      date: startDate.toString(),
```
Comment (Kheirah) [inline pages/Add/index.js:19]: You could use the .**toISOString()** method to be safe. It's the international standard for handling date and time data. Your app is probably not going to experience any issues if you don't use it, but now at least you've heard of it =)

**Review by Kheirah — APPROVED** [review]
LGTM!

File: pages/Add/index.js (hunk 2 above)
Comment (pmarx92) [reply pages/Add/index.js]: i didn't know that, thanks :)

**Pull request merged**